packages/kuma-gui/src/app/application/components/route-view/README.md

```diff
@@ -93,12 +93,10 @@ Defining these "dependencies" here has several advantages:
 4. Boolean route params are transformed correctly for the URL. In the above
    example the resulting query parameter will be the existence of `?checked`
    not `?checked=true` and `?checked=false`
-5. Route params can be used as `v-model` values. Ideally we usually use events
-   to change route properties, but for components that only support `v-model`
-   you can use the route.param as the v-model. For example `<KCheckBox
-   v-model="route.params.checked" />` will automatically sync the checkbox state
-   to the `?checked` query parameter. This aims to make it easier to maintain
-   state in the browser URL and harder not to
+5. Route params can be used as a state for input elements. In order to retrieve and update the route properties
+   we can use events. For example `<KCheckBox :model-value="route.params.checked" @change="(value) => route.update({ checked: value })" />`
+   will automatically sync the checkbox state to the `?checked` query parameter. This aims to make it easier to maintain
+   state in the browser URL and harder not to. Note that this does not work when using the two-way binding attribute `v-model`.
```

Comment:
If it was me I'd just remove point 5. I don't believe vue-router's params can be used as `v-model` values either